src/FSharpLint.Core/Prelude.fs

```diff
@@ -4,11 +4,11 @@ namespace FSharpLint.Core
 module Prelude =
 
     module Async =
-        let combine f x y = async {
-            let! x = x
-            let! y = y
-            return f x y }
+        let combine operation firstAsync secondAsync = async {
```

Comment:
@webwarrior-ws I think better names would be func job1 job2? what do you think?

Reply:
renamed to job1 an job2